src/cdk/testing/test-harnesses.md

```diff
@@ -498,8 +498,8 @@ class MyPopupHarness extends ComponentHarness {
 
 #### Accessing elements outside of the component's host element
 
-There are times when a component harness might need to access elements outside of it's corresponding
-component's host element. A good example of this is components that use the
+There are times when a component harness might need to access elements outside of its corresponding
+component's host element. Some good examples of this are the components that use the
```

Comment:
I would do
"Components that use CDK overlay serve as examples of this"

Avoids passive voice